docs/configuration/index.md

```diff
@@ -2026,8 +2026,8 @@ You can optionally only configure caching to be enabled on the Broker by setting
 See [cache configuration](#cache-configuration) for how to configure cache settings.
 
 :::info
- Note: Even if cache is enabled, for [groupBy v2](../querying/groupbyquery.md#strategies) queries, segment level cache do not work on Brokers.
- See [Differences between v1 and v2](../querying/groupbyquery.md#differences-between-v1-and-v2) and [Query caching](../querying/caching.md) for more information.
+ Note: Even if cache is enabled, for [groupBy](../querying/groupbyquery.md) queries, segment level cache does not work on Brokers.
+ See [query caching](../querying/caching.md) for more information.
```

Comment:
```suggestion
 See [Query caching](../querying/caching.md) for more information.
```

Reply:
I see lots of other links like this.... but why?

Reply:
i mean, its in the middle of a sentence, why would we uppercase it

Reply:
FWIW, personally I find the suggested style to be better :)

Reply:
I guess like... I disagree, but we have a lot of other links doing that too.

Like take this example:
<img width="904" alt="Screenshot 2023-08-22 at 9 08 33 PM" src="https://github.com/apache/druid/assets/1577461/1e43e967-3ccc-4848-b4e1-e100ba59341f">

Why should the A in "Advance groupBy v2 configurations"  and the M in "Memory tuning and resource limits" be uppercase? 

Is this a matter of perspective? Like it bothers me because these inline links are just some text that is part of a sentence that happen to be clickable. But i guess another perspective might be that this is some proper section title and it should match as closely as possible? I still don't really personally like it, but i changed it because a handful of other links are like this on this and other pages (not all of them though...)

Reply:
For me, it seems better because upper-casing highlights that part of the text. But yes, in the end, its a matter of perspective. I don't have an objective argument for either or against.

Reply:
@clintropolis In [this specific example](https://github.com/apache/druid/pull/14866#pullrequestreview-1587410573), [Query caching](https://druid.apache.org/docs/latest/querying/caching) is the title of the page and must be referenced using sentence case.

Reply:
IMO @ektravel's perspective makes sense. If the link text is a title of a page and is referring to the page itself then capitalizing the first letter seems good. It emphasizes that we're referring to a documentation page rather than a concept.

To me this is correct (the text refers to the page being linked to):

```
See [Query caching](../querying/caching.md) for more information.
```

And this is also correct (the text refers to a concept, not the page being linked to):

```
It is important to set up [query caching](../querying/caching.md) properly for best performance.
```

Btw, the following is also IMO correct, but awkward and I'd avoid it in favor of the first example. Here "query caching" is a common noun acting as an adjective; it's not referring to the page being linked to, so it shouldn't be capitalized:

```
See the [query caching](../querying/caching.md) documentation for more information.
```